src/Blazor.Auth0.ServerSide/AuthenticationService.cs

```diff
@@ -70,7 +71,7 @@ private set
         public AuthenticationService(
             HttpClient httpClient,
             IHttpContextAccessor httpContextAccessor,
-            IUriHelper uriHelper,
+            NavigationManager uriHelper,
```

Comment:
Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?